Formula/libmatheval.rb

```diff
@@ -0,0 +1,40 @@
+class Libmatheval < Formula
+  desc "Library to parse and evaluate symbolic expressions input as text."
+  homepage "https://www.gnu.org/software/libmatheval/"
+  url "https://ftpmirror.gnu.org/libmatheval/libmatheval-1.1.11.tar.gz"
+  sha256 "474852d6715ddc3b6969e28de5e1a5fbaff9e8ece6aebb9dc1cc63e9e88e89ab"
+
+  depends_on "bdw-gc" => :linked
+  depends_on "bison" => :build
+  depends_on "guile"
+
+  def install
+    # The configure script adds the Guile link directives in the wrong order.
+    ENV.append "LDFLAGS", `guile-config link`
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    # "tests" depends on deprecated Guile features.
```

Comment:
This also seems problematic. A vanilla "make install" should work, and it would be nice to be able to run the tests. Has the fact that the tests rely on deprecated features been reported upstream?